### PR TITLE
More descriptive exception message

### PR DESCRIPTION
**Pull request by jongotlin**

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Nice addition @jongotlin, thanks.

It seems to me a common practice that in such exceptions when the type is object, the FQCN is given as type instead of "object". WDYT?

**Comment by jongotlin** [comment]
I had to look in the manual when not using annotation and the example uses only the class name. But if you want me to change I'm happy to do so.
https://phpunit.de/manual/4.8/en/phpunit-book.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest3.php

**Comment by sagikazarmark** [comment]
I think you misunderstood me. I meant [this scenario](https://github.com/moneyphp/money/pull/194/files#diff-a3b23eb259814cf0992e941cd03fd471R132) should return `stdClass` as "type", not `object`.

**Comment by jongotlin** [comment]
Ah. Isn't that the case when you're using `get_class`? `gettype` doesn't care about what object it is.

**Comment by jongotlin** [comment]
Or do mean you want me to check if its an object, return its fqcn?

**Comment by sagikazarmark** [comment]
> Or do mean you want me to check if its an object, return its fqcn?

Exactly.

**Comment by jongotlin** [comment]
Got it :-) 
Even better

File: src/Money.php [hunk 1]
```diff
@@ -244,14 +244,17 @@ public function subtract(Money $subtrahend)
     /**
      * Asserts that the operand is integer or float.
      *
-     * @param float|int|string $operand
```
Comment (frederikbosch) [inline src/Money.php]: I think an operand of '1.5' should be allowed.

**Comment by frederikbosch** [comment]
Looks good, one remark.

File: src/Money.php (hunk 1 above)
Comment (jongotlin) [inline src/Money.php]: You're right. My misstake.

**Comment by frederikbosch** [comment]
Thanks @jongotlin

**Pull request merged**